workflow/packer/packer.go

```diff
@@ -48,7 +57,20 @@ func getSize(wf *wfv1.Workflow) (int, error) {
 	return len(nodeContent), nil
 }
 
+// getNodeStatusSize return the workflow node status json string size
+func getNodeStatusSize(wf *wfv1.Workflow) (int, error) {
+	nodeContent, err := json.Marshal(wf.Status.Nodes)
+	if err != nil {
+		return 0, err
```

Comment:
May need to check for specific errors? Does size of 0 make sense for all errors?

Reply:
This is the same as getSize(), I have checked all callers, this error will be handled.